react/features/base/react/components/web/InlineDialogFailure.js

```diff
@@ -0,0 +1,72 @@
+import AKButton from '@atlaskit/button';
+import React, { Component } from 'react';
+
+import { translate } from '../../../i18n';
+
+declare var interfaceConfig: Object;
+
+/**
+ * Inline dialog the represents a failure and allows a retry.
+ */
+class InlineDialogFailure extends Component {
```

Comment:
But then again it's also a failure dialog for an inline dialog so I don't really know if changing names is right...